.agent/skills/test-performance/SKILL.md

```
@@ -49,151 +28,78 @@ public class MyBenchmark
+- **Document baselines** to track regressions
+- Reports generated in `Reports/BenchmarkDotNet.Artifacts/` (HTML, MD, JSON)
 
 ---
 
 ## Related Skills
 
 - [overview-drn-testing.md](../overview-drn-testing/SKILL.md) - Testing philosophy
-- [drn-testing.md](../drn-testing/SKILL.md) - Framework.Testing
-
----
+- [drn-testing.md](../drn-testing/SKILL.md) - Framework.Testing
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)